### ISSUE TITLE
Parameter `normalize` in linear models

### ISSUE THREAD

**Issue by alex-hse-repository** [tinkoff-ai/etna#660]
### 🐛 Bug Report

FutureWarning: 'normalize' was deprecated in version 1.0 and will be removed in 1.2. Please leave the normalize parameter to its default value to silence this warning. The default behavior of this estimator is to not do any normalization. If normalization is needed please use sklearn.preprocessing.StandardScaler instead.

### Expected behavior

I guess we need to remove the parameter `normalize` from the constructor of linear models

### How To Reproduce

```
from etna.datasets import TSDataset
from etna.pipeline import Pipeline
from etna.models import LinearPerSegmentModel
import pandas as pd
from etna.transforms import SegmentEncoderTransform

df = pd.DataFrame({"segment":"a",
                   "target":[1,2,3,4,5,6,7],
                   "timestamp": pd.date_range("2000-01-01", "2000-01-07")})
ts = TSDataset(df=TSDataset.to_dataset(df), freq="D")
ts.fit_transform([SegmentEncoderTransform()])
model = LinearPerSegmentModel()
model.fit(ts)
```

### Checklist

- [X] Bug appears at the latest library version
